lib/modules/commentTools.js

```diff
@@ -259,6 +259,12 @@ module.go = () => {
 		});
 	}
 
+	// slideEditBar on comments only
+	$body.on('input', 'form[id^=\'commentreply\'] textarea', function() {
```

Comment:
I feel like it should be possible to do this up front in `attachEditorToUsertext` (i.e. never add the handler) instead of covering up the issue later. It will probably be necessary to change [this line](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/41f4e2f757e9d6d2bd1b5863b88d92847130806c/lib/modules/commentTools.js#L606) to `return cachedEditBar.clone();` so the first call to `makeEditBar` doesn't return the cached one.

Unless this happens because reddit clones reply textareas from a textarea that we've already modified. Then things get more _interesting_...